### ISSUE TITLE
DbxClientV2.refreshAccessToken() returns incorrect DbxRefreshResult.getExpiresAt()

### ISSUE THREAD

**Issue by winstonxie3000** [dropbox/dropbox-sdk-java#304]
SDK version: 3.1.5

 ```@Override
        public DbxRefreshResult refreshAccessToken() throws DbxException {
            credential.refresh(this.getRequestConfig());
            return new DbxRefreshResult(credential.getAccessToken(), credential.getExpiresAt());
        }
```

this method pass the expiresAt to expiresIn. And when DbxRefreshResult.getExpiresAt() it calculate by "issueTime + expiresIn * 1000;" However, the value of expiresIn is incorrect now.

**Comment by greg-db** [comment]
Thanks, we'll look into it.

**Comment by luigiafassina** [comment]
any news?
I'm trying to use this sdk but I'm not able to get thumbnail because my token is expired. When I try to refresh it, I received
dbxRefreshResult.getExpiresAt() 1627284964491572 so something in microseconds that corresponds to Mon Jul 26 2021 07:36:04: if I use the new access token soon, there are no problem, but when I use it after few hours it is still expired (even if the date is in the future).

**Comment by greg-db** [comment]
@luigiafassina No, unfortunately I don't have an update on this. I'll check in with the team.

**Comment by luigiafassina** [comment]
thanks a lot for the fix !! 😃 
what should happen now? I mean, is it in plan a new sdk before september?

**Comment by greg-db** [comment]
@luigiafassina Yes, [the fix](https://github.com/dropbox/dropbox-sdk-java/pull/322) has been merged, so that will be included in the next release of the SDK.

**Comment by luigiafassina** [comment]
hi, do you have in plan to release a new SDK (that include this fix) soon ? more or less when ?

**Comment by greg-db** [comment]
@luigiafassina I don't have a timeline for the next release, but I'll follow up here once I have any news.

**Comment by greg-db** [comment]
This is fixed in release [v4.0.1](https://github.com/dropbox/dropbox-sdk-java/releases/tag/v4.0.1).

**Comment by luigiafassina** [comment]
Thanks a lot!!
